cpp/include/cudf/fixed_point/fixed_point.hpp

```diff
@@ -139,7 +139,11 @@ class fixed_point {
         return detail::shift<Rad>(static_cast<U>(_value), detail::negate(_scale));
     }
 
-    auto get() const noexcept {
+    auto to_int32() const noexcept {
```

Comment:
No need for auto here, right?  Also, why are these needed if you have the explicit conversion operator?

Reply:
@harrism @jrhemstad I just want to double check before I make this change that this is what we want. Most of the `fixed_point` implementations a researched over the last couple days most require a `static_cast<int/float>()` for converting. I will remove these methods if we agree we don't need them. That is my vote.

Reply:
If no objection, i will remove tomorrow.

Reply:
Done.